floor_generator/lib/processor/view_processor.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/type.dart';
+import 'package:floor_annotation/floor_annotation.dart' as annotations;
+import 'package:floor_generator/misc/annotations.dart';
+import 'package:floor_generator/misc/constants.dart';
+import 'package:floor_generator/misc/type_utils.dart';
+import 'package:floor_generator/processor/error/view_processor_error.dart';
+import 'package:floor_generator/processor/field_processor.dart';
+import 'package:floor_generator/processor/processor.dart';
+import 'package:floor_generator/value_object/field.dart';
+import 'package:floor_generator/value_object/view.dart';
+
+class ViewProcessor extends Processor<View> {
```

Comment:
I didn't put `_getName()` in the `QueryableProcessor` because it is pretty dependent on the Type (`View` vs `Entity`). But for the rest that's a pretty good idea.